### PR TITLE
Code cleanup/PR feedback for Messages

### PR DESCRIPTION
**Pull request by bruce-glazier**
## Changes

This PR is almost only renames and code cleanup with the exception of tweaking the message timestamp abbreviations a bit.

### CODE REVIEW

File: src/screens/MessageScreen.tsx [hunk 1]
```diff
@@ -9,13 +9,8 @@ import { t } from 'i18next';
...
 import { useMyMessages } from '../hooks/useMyMessages';
-import {
-  format,
-  differenceInMinutes,
-  differenceInHours,
-  differenceInDays,
-  differenceInSeconds,
-} from 'date-fns';
+import { format, differenceInDays } from 'date-fns';
+import dayjs from 'dayjs';
```
Comment (aecorredor) [inline src/screens/MessageScreen.tsx:13]: My two cents here are that it'd be great if we could consolidate on a single date library for the SDK. I think that means that keeping your previous code that manually calculated the "from now" format while only keeping `date-fns` looks better than having both `date-fns` and `dayjs` at the same time just for `dayjs`'s format helper.

Comment (bruce-glazier) [reply src/screens/MessageScreen.tsx:13]: I wonder if we should just swap date-fns with dayjs. I think date-fns doesn't have nearly as much utility and we are not too deep with date-fns yet.

Comment (bruce-glazier) [reply src/screens/MessageScreen.tsx:13]: I think for now I'll backpedal on the dayjs decision and use the function I initially created.

Comment (swain) [reply src/screens/MessageScreen.tsx:13]: +1, sounds good. If it does come around, I'm personally in favor of `dayjs`. I find it's much more ergonomic than `date-fns`.

Comment (bruce-glazier) [reply src/screens/MessageScreen.tsx]: Done, switched back to a function.

File: src/screens/MessageScreen.tsx [hunk 2]
```diff
@@ -150,6 +179,16 @@ const defaultStyles = createStyles('MessageScreen', (theme) => {
     },
     badgeView: {
       alignSelf: 'center',
+      paddingRight: 0,
+      marginRight: -10,
+    },
+    badgeColor: {
+      enabled: {
+        backgroundColor: theme.colors.primary,
```
Comment (bruce-glazier) [inline src/screens/MessageScreen.tsx:187]: A consumer of this can set this color to transparent if they don't want the badge.

Comment (sternetj) [reply src/screens/MessageScreen.tsx:187]: Or they can do `badgeView.display: 'none'` That's my go to for hiding content.

**Review by sternetj — APPROVED** [review]

**Pull request merged**